Index.html

```diff
@@ -33,6 +34,154 @@ <h2 class="subtitulo">I&apos;am a Software Developer</h2>
+            <div class="list_programs2">
+              <ul>
+                <li<a><img class="htm"src="./images/html2.svg"></a></li> 
+                <li<a><img class="bot"src="./images/boot2.svg"></a></li>
+                <li<a><img class="ruby"src="./images/ruby2.svg"></a></li>
+              </ul>
+            </div>
+              <button><a class="boton2">See Project</a></button> 
+            </div>
+    
+            <div class="projects">
+              <h1 class="tituloPAPD">Profesional Art Printing Data</h1>
+              <p class="parrafo2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+              <div class="list_programs2">
+                <ul>
+                  <li<a><img class="htm"src="./images/html2.svg"></a></li> 
+                  <li<a><img class="bot"src="./images/boot2.svg"></a></li>
+                  <li<a><img class="ruby"src="./images/ruby2.svg"></a></li>
+                </ul>
+              </div>
+                <button><a class="boton2">See Project</a></button> 
+              </div>
+
+              <div class="projects">
+                <h1 class="tituloPAPD">Profesional Art Printing Data</h1>
+                <p class="parrafo2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                <div class="list_programs2">
+                  <ul>
+                    <li<a><img class="htm"src="./images/html2.svg"></a></li> 
+                    <li<a><img class="bot"src="./images/boot2.svg"></a></li>
+                    <li<a><img class="ruby"src="./images/ruby2.svg"></a></li>
+                  </ul>
+                </div>
+                  <button><a class="boton2">See Project</a></button> 
+                </div>
+        
+                <div class="projects">
+                  <h1 class="tituloPAPD">Profesional Art Printing Data</h1>
+                  <p class="parrafo2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                  <div class="list_programs2">
+                    <ul>
+                      <li<a><img class="htm"src="./images/html2.svg"></a></li> 
+                      <li<a><img class="bot"src="./images/boot2.svg"></a></li>
+                      <li<a><img class="ruby"src="./images/ruby2.svg"></a></li>
+                    </ul>
+                  </div>
+                    <button><a class="boton2">See Project</a></button> 
+                  </div>
```

Comment:
- [ ] Please consider putting all your project cards in a single container. Also, align the contents of the single container using a CSS grid as demanded by the project requirements.